src/style.css

```diff
@@ -0,0 +1,110 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 30px 14px 14px;
+  margin: 0;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
+}
+
+.card__description {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0 0 5px;
+}
+
+.card__code {
+  font-size: 11px;
+  line-height: 13px;
+  margin: 0 0 15px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 15px;
+  margin: 0 0 20px;
+}
+
+.card__stars {
+  display: flex;
+  align-items: center;
+}
+
+.star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--5 .star,
+.stars--1 .star:first-child,
+.stars--2 .star:nth-child(-n+2),
+.stars--3 .star:nth-child(-n+3),
+.stars--4 .star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__review {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__value {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.card__caption {
+  font-size: 11px;
+  line-height: 13px;
+  margin: 0;
+}
+
+.card__price {
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 19px;
+  margin: 0 0 20px;
+}
+
+.card__button {
+  width: 170px;
+  height: 40px;
+  line-height: 40px;
+  text-align: center;
+  background-color: #00acdc;
+  border-radius: 5px;
+  color: white;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 14px;
+}
+
+.card__button:hover {
+  background-color: white;
+  color: #00acdc;
+  border: 1px solid #00acdc;
```

Comment:
at this moment when you hovered on your link - border appears and adds 1px, because of box-sizing: border-box; width and height of your button is recalculating, and text could change his position, so its better if you write border property inside the button.

Reply:
done !